tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -0,0 +1,350 @@
+# Copyright 2023 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import math
+
+from rptest.services.cluster import cluster
+from rptest.tests.redpanda_test import RedpandaTest
+from ducktape.tests.test import TestContext
+from rptest.services.producer_swarm import ProducerSwarm
+from rptest.clients.rpk import RpkTool
+from rptest.services.redpanda_cloud import AdvertisedTierConfigs, CloudTierName
+from rptest.services.redpanda import (SISettings, RedpandaServiceCloud)
+from rptest.services.openmessaging_benchmark import OpenMessagingBenchmark
+from rptest.services.openmessaging_benchmark_configs import \
+    OMBSampleConfigurations
+
+KiB = 1024
+MiB = KiB * KiB
+GiB = KiB * MiB
+minutes = 60
+hours = 60 * minutes
+
+
+def get_globals_value(globals, key_name, default=None):
+    _config = {}
+    if RedpandaServiceCloud.GLOBAL_CLOUD_CLUSTER_CONFIG in globals:
+        # Load needed config values from cloud section
+        # of globals prior to actual cluster creation
+        _config = globals[RedpandaServiceCloud.GLOBAL_CLOUD_CLUSTER_CONFIG]
+    return _config.get(key_name, default)
+
+
+class OMBValidationTest(RedpandaTest):
+    def __init__(self, test_ctx: TestContext, *args, **kwargs):
+        self._ctx = test_ctx
+        # Get tier value
+        cloud_tier_str = get_globals_value(self._ctx.globals,
+                                           "config_profile_name",
+                                           default="tier-1-aws")
+        cloud_tier = CloudTierName(cloud_tier_str)
+        extra_rp_conf = None
+        num_brokers = None
```

Comment:
How does `num_brokers` get set if not in docker?

Reply:
If it's a cloud cluster its never set, otherwise;

```python
          if num_brokers is None and cloud_tier is None:                                                                                                                                      
              # Default to a 3 node cluster if sufficient nodes are available, else                                                                                                           
              # a single node cluster.  This is just a default: tests are welcome                                                                                                             
              # to override constructor to pass an explicit size.  This logic makes                                                                                                           
              # it convenient to mix 3 node and 1 node cases in the same class, by                                                                                                            
              # just modifying the @cluster node count per test.                                                                                                                              
              if test_context.cluster.available().size() >= 3:                                                                                                                                
                  num_brokers = 3                                                                                                                                                             
              else:                                                                                                                                                                           
                  num_brokers = 1                                                                                                                                                             
```